v1/lib/server/liveReloadServer.js

```diff
@@ -9,10 +9,9 @@ const gaze = require('gaze');
 const tinylr = require('tiny-lr');
 const readMetadata = require('./readMetadata.js');
 
-let reloadScriptUrl;
-
 function start(port) {
   process.env.NODE_ENV = 'development';
+  process.env.PORT = port;
```

Comment:
This env variable is already being used by the webserver, so we can't call it `process.env.PORT`. Maybe `process.env.LIVERELOAD_PORT`?

Reply:
Thanks for the catch!